frame/oracle/src/lib.rs

```diff
@@ -981,6 +989,14 @@ pub mod pallet {
 					PrePrices::<T>::remove(asset_id);
 
 					Self::handle_payout(&pre_prices, price, asset_id, &asset_info)?;
+
+					// Emit `PriceChanged` event when PICA or PABLO prices have changed.
+					if price != last_price &&
+						vec![CurrencyId::PICA, CurrencyId::PBLO]
```

Comment:
We can set this as an attribute/flag of `asset_info` and based on that emit the event rather than hard coding

Reply:
The consideration is whether we want the event emission for a particular asset type to be configurable with an extrinsic call or runtime upgrade. I think in this case as assets are configured with a extrinsic it makes sense to use the same way to configure whether events are emitted for them. So let's go with a new flag on `asset_info`.